epub33/core/index.html

```diff
@@ -8071,12 +8071,39 @@ <h3>Navigation Document Overlays</h3>
 				</div>
 			</section>
 		</section>
-		<!-- <section id="sec-privacy">
-			<h2>Privacy</h2>
-			<div class="ednote">
-				<p>The working group will address issues related to privacy of EPUB Publications in a future draft.</p>
+		<section id="sec-accessibility" class="informative">
+			<h2>Accessibility</h2>
+
+			<p>EPUB 3 is built upon the Open Web Platform expressly so that it can leverage the structure, semantics
+				and, by extension, accessibility built into its underlying technologies.</p>
+
+			<p>The requirements and practices for creating accessible web content have already been documented in the
+				W3C's <a href="https://www.w3.org/TR/wcag/">Web Content Accessibility Guidelines (WCAG)</a> [[!WCAG]].
+				These guidelines also form the basis for defining accessibility in EPUB Publications.</p>
```

Comment:
I presume it should be [[WCAG]] now; I guess that is why respec is a bit unhappy and puts some sort of an underling underneath...

Reply:
Ya, I should probably try rendering the document again when slapping an informative label on something... ;)